bbc/simorgh
# NEWSWORLDSERVICE-1615: Use fixture data from local filesystem for topic pages

Resolves JIRA NEWSWORLDSERVICE-1615

Overall changes
======
Enables the use of local fixture data for topics - no need to fetch from the BFF

Code changes
======

- Enables fetching the topic data from the local filesystem as well as the BFF
- Updates the path to the topic data file, including support for variants

Testing
======
1. Ensure http://localhost:7080/kyrgyz/topics/cvpv9djp9qqt.json returns data
2. Ensure http://localhost:7080/kyrgyz/topics/cvpv9djp9qqt renders as expected, and no call to the BFF is made
3. Ensure http://localhost:7080/kyrgyz/topics/cvpv9djp9qqt?renderer_env=test renders as expected, and data is fetched via the BFF

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
